### ISSUE TITLE
[Catalog] Publish Configuring Prometheus

### ISSUE THREAD

**Issue by theBeginner86** [meshery/meshery#9626]
### Current Behavior

Publish [Configuring Prometheus](https://github.com/prometheus-operator/prometheus-operator/tree/main/example) to [Meshery Catalog](https://meshery.io/catalog).

### Environment

- **Host OS:** Mac Linux Windows
- **Platform:** Docker or Kubernetes
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

**Comment by professorabhay** [comment]
Hey, @theBeginner86! I want to work on this. 
Are there any resources that can help?

**Comment by yash37158** [comment]
Here you go @professorabhay

**Comment by leecalcote** [comment]
Removing @professorabhay's assignment until he is prepared to undertake the issue.

**Comment by theBeginner86** [comment]
> I'll be taking on this issue.

Would you please confirm if you are comfortable publishing a design? Are you successful running Meshery locally?

**Comment by akkshitgupta** [comment]
I've successfully set up the local environment @theBeginner86. I am not able to figure out how to publish a design. I am trying to understand the in and outs of the publish process but could not figure out how to obtain that `random string` used in the `.yaml` file and confused whether we can take that sample code as it belongs to others.  
![image](https://github.com/meshery/meshery/assets/96991785/0b774757-b1cb-4086-bee6-4782def062c7)

**Comment by yash37158** [comment]
Got it, @akkshitgupta! I recommend checking out the documentation on publishing a design https://docs.meshery.io/extensions/publishing-a-design

**Comment by AdarshRawat1** [comment]
> Would you please confirm if you are comfortable publishing a design? Are you successful running Meshery locally?

I'm comfortable in publishing a design , I'm a bit occupied with my end term exams. I'll open a PR soon .

**Comment by akkshitgupta** [comment]
@yash37158 I went through the docs you suggested.
Here, the docs have this statement and I am not the author. 
> Note: To publish a design you must be the author of the design.
Can I still publish the design the issue contains??

**Comment by yash37158** [comment]
Yes @akkshitgupta, ensure that the design is authentically yours, created from scratch. This will make it easier to verify the authenticity of your design.

**Comment by AdarshRawat1** [comment]
@akkshitgupta , It appears you've begun working on the issue. If that's the case, please inform me to prevent any duplication of efforts.
P.s. Then I can then shift my focus to resolving a different issue.

**Comment by akkshitgupta** [comment]
@AdarshRawat1 I was going through the issue but didn't get much understanding about it. You can work on this issue. Thank you

**Comment by theBeginner86** [comment]
> @yash37158 I went through the docs you suggested. Here, the docs have this statement and I am not the author.
> 
> > Note: To publish a design you must be the author of the design.
> 
> Can I still publish the design the issue contains??

Yes, you can publish. "Note: To publish a design you must be the author of the design." means that the Meshery Design you are trying to publish must be owned by you.

Basically, it means you should see your avatar in the design info modal.
Eg:
<img width="890" alt="Screenshot 2023-12-27 at 5 03 16 PM" src="https://github.com/meshery/meshery/assets/73700530/e2c9fe68-0d64-426e-8527-f7d74b9f8ce4">

**Comment by AdarshRawat1** [comment]
> @AdarshRawat1 I was going through the issue but didn't get much understanding about it. You can work on this issue. Thank you

@akkshitgupta  Noted!

**Comment by saikat709** [comment]
@AdarshRawat1 Are you working on this?

**Comment by AdarshRawat1** [comment]
> @AdarshRawat1 Are you working on this?

Go ahead , you can work on this.
